airbyte-webapp/src/views/Connection/ConnectionForm/ConnectionFormFields.tsx

```diff
@@ -0,0 +1,184 @@
+  title?: React.ReactNode;
+}
+
+const TryArrow = styled(FontAwesomeIcon)`
+  margin: 0 10px -1px 0;
+  font-size: 14px;
+`;
+
+export const StyledSection = styled.div`
+  padding: 20px 20px;
+  display: flex;
+  flex-direction: column;
+  gap: 15px;
+
+  &:not(:last-child) {
+    box-shadow: 0 1px 0 rgba(139, 139, 160, 0.25);
+  }
+`;
+
+export const FlexRow = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-start;
+  align-items: flex-start;
+  gap: 10px;
+`;
+
+export const LeftFieldCol = styled.div`
+  flex: 1;
+  max-width: 640px;
+  padding-right: 30px;
+`;
+
+export const RightFieldCol = styled.div`
+  flex: 1;
+  max-width: 300px;
+`;
+
+export const LabelHeading = styled(H5)`
+  line-height: 16px;
+  display: inline;
+`;
+
+export const ConnectorLabel = styled(ControlLabels)`
+  max-width: 328px;
+  margin-right: 20px;
+  vertical-align: top;
+`;
+
+const NamespaceFormatLabel = styled(ControlLabels)`
+  flex: 5 0 0;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+`;
+
+export const FormContainer = styled(Form)`
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+`;
+
+export const Section: React.FC<React.PropsWithChildren<SectionProps>> = ({ title, children }) => (
+  <Card>
+    <StyledSection>
+      {title && <H5 bold>{title}</H5>}
+      {children}
+    </StyledSection>
+  </Card>
+);
+
+interface ConnectionFormFieldsProps {
+  className?: string;
+  values: ValuesProps | FormikConnectionFormValues;
+  isSubmitting: boolean;
+}
+
+export const ConnectionFormFields: React.FC<ConnectionFormFieldsProps> = ({ className, values, isSubmitting }) => {
+  const { mode, onRefreshSourceSchema } = useConnectionFormService();
+  const { formatMessage } = useIntl();
+
+  return (
+    <FormContainer className={className}>
+      <Section title={<FormattedMessage id="connection.transfer" />}>
+        <ScheduleField />
+      </Section>
+      <Card>
+        <StyledSection>
+          <H5 bold>
+            <FormattedMessage id="connection.streams" />
+          </H5>
+          <span style={{ pointerEvents: mode === "readonly" ? "none" : "auto" }}>
+            <Field name="namespaceDefinition" component={NamespaceDefinitionField} />
+          </span>
+          {values.namespaceDefinition === NamespaceDefinitionType.customformat && (
+            <Field name="namespaceFormat">
+              {({ field, meta }: FieldProps<string>) => (
+                <FlexRow>
+                  <LeftFieldCol>
+                    <NamespaceFormatLabel
+                      nextLine
+                      error={!!meta.error}
+                      label={<FormattedMessage id="connectionForm.namespaceFormat.title" />}
+                      message={<FormattedMessage id="connectionForm.namespaceFormat.subtitle" />}
+                    />
+                  </LeftFieldCol>
+                  <RightFieldCol style={{ pointerEvents: mode === "readonly" ? "none" : "auto" }}>
```

Comment:
I know this is still WIP -- we can/should just use classnames and pull out this repeated logic as part of that.  (I see this is on every input, as it should be :) )